reports/California.md

```diff
@@ -1286,6 +1286,20 @@ id: ca-sandiego-6
 * https://twitter.com/TheActivatedPod/status/1309169875064061954
 
 
+### Police shove protesters with bikes and arrest them | September 25th
+
+Police pin protesters against a wall, shoving them with bikes. Protesters then push back, knocking one officer to the ground.
+
+tags: bike, protester, shove, push, arrest, baton
```

Comment:
Wait there's arrest and baton in this? Can you add it to the description?

Reply:
yea no arrests not sure why I put that. Saw baton use at 0:40. Is it a stretch? wasn't a direct hit but it hit a sign